source/_cookbook/SSL guide.md

```diff
@@ -0,0 +1,570 @@
+```
+
+At the bottom of the file add the following lines:
+ 
+```text
+interface wlan0 <----- or the interface you just wrote down.
+
+static ip_address=192.168.0.200/24  <---- the IP address you just wrote down with a '/24' at the end
+static routers=192.168.0.1      <---- Your router's IP address
+static domain_name_servers=192.168.0.1 <---- Your router's IP address
+```
+
+It is important to note that the first three bits of your static IP address and your router's IP address should be the same, eg:
+
+```text
+Router: 192.168.0.1
+
+Yes
+HA IP: 192.168.0.200
+
+No
+HA IP: 192.175.96.200
+```
+
+Press Ctrl + x to close the editor, pressing Y to save the changes when prompted
+
+Reboot your pi:
+
+```bash
+$ sudo reboot
+```
+
+When it comes back up check that you can SSH in to it again on the IP address you wrote down.
+
+Make sure HA is running and access it via the local network by typing the IP address and port number in to the browser:
+
+```text
+http://192.168.0.200:8123.
+```
+
+All working?  Hooray!  You now have a static IP.  This will now always be your internal IP address for your HA device.  This will be known as YOUR-HA-IP for the rest of this guide.
+
+
+
+#### {% linkable_title 2 - set up port forwarding without SSL and test connection %}
+
+Log in to your router's configuration pages and find the port forwarding options.  This bit is hard to write a guide for because each router has a different way of presenting these options.  Searching google for "port forwarding" and the name of your router may help.  When you find it you will likely have options similar to:
+
+Service name - Port Range -  Local IP -  Local Port - Protocol
+
+You may also have other options (like 'source IP'), these can usually be left blank or in their default state.
+
+Set the port forwarding to:
+
+```text
+Service name - ha_test
+Port Range - 8123
+Local IP - YOUR-HA-IP
+Local Port - 8123
+Protocol - Both
+```
+
+Then save the change.  On my router you have to fill these values in, then press an 'add' button to add the new rule to the list, then save the changes.  All routers have a different interface, but you must ensure that these rules are saved at this point.  If you are unsure, you can reboot the router and log back in, if the rule is present it was saved, if not, it wasn't!
+
+Once you have saved this rule, go to your browser, and go to:
+
+```text
+https://whatismyipaddress.com/
+```
+
+This will tell you your current external IP address
+
+Type the external IP address in to the url bar with http:// in front and :8123 after like so (12.12.12.12 is my example!):
+
+```text
+http://12.12.12.12:8123
+```
+
+Can you see your HA instance?  Awesome! If not, your router may not support ' loopback' - try the next step anyway and if that works, and this one still doesn't, just remember that you cannot use loopback, so will have to use internal addresses when you're on your home network.  More on this later on if it's relevant to you.

```

Comment:
The fact that you put _loopback_ in quotes suggests that it has a meaning which a lot of users don't understand.
I guess it should either be explained or left out.